### PR TITLE
Added notify system.

### PR DESCRIPTION
**Pull request by normanjaeckel**
@FinnStutzenstein Please play around a little bit. Server side (echo) should work for now.

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
@normanjaeckel I cannot push into your branch, so please look at my branch.

You can test my first idea of the notifying users, when others edit dialogs are open (currently just the array with names, the ui follows if everything works). With logged in users, this works fine.

I have a problem with anonymous users. With every notify, I pass a `senderId`, so the receiver can answer to this user directly. First, I would like to put [this line](https://github.com/FinnStutzenstein/OpenSlides/commit/5b7335737e4695b816f303de13759383a239a97a#diff-e97f2fd30e66e33689576c55af91d05eR381) to the server, so a user cannot corrupt this and second, whats about anonymous users?
I think, because they do not have an id, we should pass the channel name, so we do have a direct replay channel. @normanjaeckel Opinions?

**Comment by normanjaeckel** [comment]
We can add the decorator `@channel_session_user` to the ws_receive_site function to get the user but I think the reply channel name is already part of the message object. Then we just need some server logic here ...

**Comment by FinnStutzenstein** [comment]
Yes, this would be great. So the server includes the reply channel. If the clients notify includes `... replyChannel: 'channelname', ...` this message is only send to this channel.

This also fixes another issue: If user A has two tabs open and user B wants to reply to a specific message from user A, currently both tabs get this notify. With giving the replychannel, just the one tab, which has send the request, gets an answer.

We also should include always the senderId. If I want to notify a user, all running instances of OS by this user should get the message.

TL;DR:
The server should add:
 - replychannel (always),
 - userId (None, if anonymous)

**Comment by normanjaeckel** [comment]
@FinnStutzenstein Done and rebased. Please rebase your own branch and test again (maybe read an correct my docstring in autoupdate.py).

**Comment by FinnStutzenstein** [comment]
@normanjaeckel This works fine! Now, I just have to do the ui and how to display the warning.

**Comment by normanjaeckel** [comment]
Ok, nice. We can merge it but then please open an issue about the following: The new code in MotionUpdateCtrl is very much. We should encapsulate it in an extra factory or service. Also some docstrings would be helpful to understand what this is doing (question query, response query, close signal ...).

**Comment by emanuelschuetze** [comment]
>  We can merge it but then please open an issue about the following

Done: #3219
